chain/near/src/trigger.rs

```diff
@@ -156,365 +159,389 @@ pub struct ReceiptWithOutcome {
-        Some(codec::Signature {
-            r#type: 0,
-            bytes: hex::decode(input)
-                .unwrap_or_else(|_| panic!("Invalid Signature value {}", input)),
-        })
-    }
-
-    struct BytesHeap {
-        api_version: graph::semver::Version,
-        memory: Vec<u8>,
-    }
-
-    impl BytesHeap {
-        fn new(api_version: graph::semver::Version) -> Self {
-            Self {
-                api_version,
-                memory: vec![],
-            }
-        }
-    }
-
-    impl AscHeap for BytesHeap {
-        fn raw_new(
-            &mut self,
-            bytes: &[u8],
-            _gas: &GasCounter,
-        ) -> Result<u32, DeterministicHostError> {
-            self.memory.extend_from_slice(bytes);
-            Ok((self.memory.len() - bytes.len()) as u32)
-        }
-
-        fn read_u32(&self, offset: u32, gas: &GasCounter) -> Result<u32, DeterministicHostError> {
-            let mut data = [std::mem::MaybeUninit::<u8>::uninit(); 4];
-            let init = self.read(offset, &mut data, gas)?;
-            Ok(u32::from_le_bytes(init.try_into().unwrap()))
-        }
-
-        fn read<'a>(
-            &self,
-            offset: u32,
-            buffer: &'a mut [std::mem::MaybeUninit<u8>],
-            _gas: &GasCounter,
-        ) -> Result<&'a mut [u8], DeterministicHostError> {
-            let memory_byte_count = self.memory.len();
-            if memory_byte_count == 0 {
-                return Err(DeterministicHostError::from(anyhow!(
-                    "No memory is allocated"
-                )));
-            }
-
-            let start_offset = offset as usize;
-            let end_offset_exclusive = start_offset + buffer.len();
-
-            if start_offset >= memory_byte_count {
-                return Err(DeterministicHostError::from(anyhow!(
-                    "Start offset {} is outside of allocated memory, max offset is {}",
-                    start_offset,
-                    memory_byte_count - 1
-                )));
-            }
-
-            if end_offset_exclusive > memory_byte_count {
-                return Err(DeterministicHostError::from(anyhow!(
-                    "End of offset {} is outside of allocated memory, max offset is {}",
-                    end_offset_exclusive,
-                    memory_byte_count - 1
-                )));
-            }
-
-            let src = &self.memory[start_offset..end_offset_exclusive];
-
-            Ok(init_slice(src, buffer))
-        }
-
-        fn api_version(&self) -> graph::semver::Version {
-            self.api_version.clone()
-        }
-
-        fn asc_type_id(
-            &mut self,
-            type_id_index: graph::runtime::IndexForAscTypeId,
-        ) -> Result<u32, HostExportError> {
-            // Not totally clear what is the purpose of this method, why not a default implementation here?
-            Ok(type_id_index as u32)
-        }
-    }
-}
+// #[cfg(test)]
```

Comment:
There should be a comment why these tests are commented out and what needs to happen to run them again

Reply:
Good catch, I've added all of these to https://github.com/graphprotocol/graph-node/issues/5079 so I can address them with mroe time. This api changed and the utility functions to create the wasm instance were not importable from test-store and wasm-runtime-test modules but I didn't want to spend more time to fix it here.